package-lock.json

```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@dashevo/dashcore-lib": {
-      "version": "0.16.4",
-      "resolved": "https://registry.npmjs.org/@dashevo/dashcore-lib/-/dashcore-lib-0.16.4.tgz",
-      "integrity": "sha512-uEkSBEI8LMoyPFzzjPghvBbkQpoVEL5yp6/r3ulAd0oTecdBH8gruwvsa9z4NlFBNO7/XcG5MVim3K8ZT7KmEA==",
+      "version": "0.16.3",
```

Comment:
Should the version in this file be `0.16.3` when the version in `package.json` is `0.16.5`? I might be missing something, but seems like maybe a discrepancy.

Reply:
yep, totally right. Will update package-lock.json as soon as we have dashcore-p2p v.1.1.9 with the new dashcore-lib 0.16.5 on npm. Waiting for this to be merged https://github.com/dashevo/dashcore-p2p/pull/11

Reply:
Published 1.1.9 on the npm